### PR TITLE
combined dagger bump

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
Build is failing ❗️

**Comment by tobiasKaminsky** [comment]
https://github.com/google/dagger/releases/tag/dagger-2.24

@ezaquarii so they removed the individual implementation, but only use HasAndroidInjector now.
But I fail to change this…

File: src/main/java/com/owncloud/android/MainApp.java [hunk 1]
```diff
@@ -110,12 +102,7 @@
  * <p>
  * Contains methods to build the "static" strings. These strings were before constants in different classes
  */
-public class MainApp extends MultiDexApplication implements
-    HasActivityInjector,
-    HasSupportFragmentInjector,
-    HasServiceInjector,
-    HasContentProviderInjector,
-    HasBroadcastReceiverInjector {
+public class MainApp extends MultiDexApplication implements HasAndroidInjector {
```
[the remark this answers is not in the document]
Comment (ezaquarii) [reply src/main/java/com/owncloud/android/MainApp.java]: Possibly not my fault.

**Comment by ezaquarii** [comment]
@tobiasKaminsky @AndyScherzinger Dagger migration is done.

File: src/main/java/com/owncloud/android/MainApp.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (AndyScherzinger) [reply src/main/java/com/owncloud/android/MainApp.java]: Such comments should be ignored for now since that is a given issue 👍

**Comment by AndyScherzinger** [comment]
I am afk but can give it a spin next week. 👍

**Comment by AndyScherzinger** [comment]
👍 Works fine for me 👍

**Comment by tobiasKaminsky** [comment]
Tested & works.
@ezaquarii thanks for fixing it. I tried nearly the same, but for an unknown reason it did not worked.

**Pull request merged**